app/api/models.js

```diff
@@ -0,0 +1,17 @@
+const { get } = require('./base')
+const { promptsApi: config } = require('../config/api')
+
+const baseUrl = config.baseUrl
+
+const listModels = async (vendor = '') => {
+  return get(`${baseUrl}/models?deploymentVendor=${vendor}`)
+}
+
+const listAllModels = async () => {
+  return get(`${baseUrl}/allmodels`)
```

Comment:
Where is this `allmodels` endpoint. This isn’t in the prompts API as far as I know.

Only vendors are ‘azure’ and ‘aws’ at the moment so could just hard code for now.

We can look at adding a ticket at a model vendor endpoint to prompts api in the next sprint.

We shouldn’t need to return all models as we will always filter at least by the vendor.

Reply:
This new endpoint is in [https://github.com/DEFRA/coreai-mcu-generation-oai/pull/14](https://github.com/DEFRA/coreai-mcu-generation-oai/pull/14)